.github/workflows/build.yml

```diff
@@ -72,7 +72,7 @@ jobs:
         npm run test:pipeline -- text-bayes-classification
     - name: install plugins from specific pipeline
       run: |
-          ./packages/cli/dist/bin/pipcook pipeline install ./example/pipelines/text-bayes-classification.json
+          ./packages/cli/dist/bin/pipcook pipeline install ./example/pipelines/chinese-poem-creation.json
```

Comment:
So why do we change the pipeline to test here?

Reply:
This makes a fresh installation with no plugins ware installed.